client/my-sites/importer/importer-blogger.jsx

```diff
@@ -48,14 +49,27 @@ class ImporterBlogger extends React.PureComponent {
 				}
 			),
 			uploadDescription: this.props.translate(
-				'Upload a {{b}}%(importerName)s export file{{/b}} to start importing into {{b}}%(siteTitle)s{{/b}}.',
+				'Upload a {{b}}%(importerName)s export file{{/b}} to start ' +
+					'importing into {{b}}%(siteTitle)s{{/b}}. If you want to know how to create the export file, ' +
+					'our {{inlineSupportLink/}} can help.',
 				{
 					args: {
 						importerName,
 						siteTitle: this.props.site.title,
 					},
 					components: {
 						b: <strong />,
+						inlineSupportLink: (
+							<InlineSupportLink
+								supportPostId={ 66764 }
+								supportLink={
+									'https://en.support.wordpress.com/import/coming-from-blogger/#import'
+								}
+								text={ importerName + ' ' + this.props.translate( 'import guide' ) }
+								showText={ true }
```

Comment:
Tiny nit,  `={ true }` can be omitted, `showText` alone will be implicitly `true`. Not a blocker ofc!

Reply:
Excellent, thanks!